src/frontend/apps/lti_site/components/DashboardVOD/index.spec.tsx

```diff
@@ -30,24 +30,30 @@ jest.mock('components/VideoWidgetProvider', () => ({
 }));
 
 const languageChoices = [
-  { label: 'English', value: 'en' },
-  { label: 'French', value: 'fr' },
+  { display_name: 'English', value: 'en' },
+  { display_name: 'French', value: 'fr' },
 ];
 
 describe('<DashboardVOD />', () => {
   beforeEach(() => {
     useJwt.setState({
       jwt: 'some token',
     });
+    fetchMock.mock({
```

Comment:
I don't think that it is correctly mocked, when I launch the test I get that:
console.warn Unmatched OPTIONS to /api/timedtexttracks/